packages/typescript/ai-fal/tests/utils.test.ts

```
@@ -0,0 +1,149 @@
+vi.mock('@fal-ai/client', () => ({
+  fal: {
+    config: vi.fn(),
+  },
+}))
+
+import {
+  configureFalClient,
+  deriveAudioContentType,
+  extractUrlExtension,
+  generateId,
+} from '../src/utils'
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)